Expecto.Hopac/Expecto.Hopac.fsproj

```diff
@@ -11,6 +11,9 @@
   <ItemGroup>
     <ProjectReference Include="..\Expecto\Expecto.fsproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
```

Comment:
this change was to address a build error when building this project against net461. It seems that System.Threading.Tasks wasn't included by default somewhere.